catalog/ui/search-ui/standard/src/main/webapp/js/controllers/openlayers.controller.js

```diff
@@ -115,18 +115,17 @@ define(['underscore',
             setupEvents: function () {
                 var controller = this;
                 this.mapViewer.on('click', function(event) {
-                    var feature;
-
-                    controller.mapViewer.forEachLayerAtPixel(event.pixel, function (layer) {
-                        feature = controller.mapViewer.forEachFeatureAtPixel(event.pixel,
-                            function(feature) {
-                                controller.trigger('click:left', feature);
-                            }, this, function(testLayer) {
-                                if (testLayer === layer) {
-                                    return true;
-                                }
-                            });
-                    });
+                    controller.mapViewer.forEachFeatureAtPixel(event.pixel,
```

Comment:
After we detect a metacard and trigger the click event, we return true which will stop feature detection: http://openlayers.org/en/v3.2.1/apidoc/ol.Map.html#forEachFeatureAtPixel